### PR TITLE
CASMMON-129 Added Grafana_Dashboards_by_Component.md documentation

### PR DESCRIPTION
**Pull request by mantoanibroadwing**
Added Grafana Dashboard documentation

Update:
main backport: https://github.com/Cray-HPE/docs-csm/pull/1858
1.0 backport: https://github.com/Cray-HPE/docs-csm/pull/1925

### CODE REVIEW

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:2]: This is not really a heading. This should just be body text. 
```suggestion

This document show the dashboards that are available for each component in Grafana.
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:10]: Grafana should be capitalized.
```suggestion
**NOTE:** There are some known issues around some of the Grafana dashboards; those are explained in [Troubleshoot_Grafana_Dashboard.md](./Troubleshoot_Grafana_Dashboard.md).
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:27]:
```suggestion
### CoreDNS
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:31]:
```suggestion
### ETCD
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:44]:
```suggestion
### Kea
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:48]:
```suggestion
### Kubernetes
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:15]: Line with Dashboard, since this list is already for Ceph, no need to include it on each item
```suggestion
- Cluster
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:16]: Similarly, Ceph should be removed from all of the other items in this list.

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:38]: As with Ceph, Istio should be removed from all of the items in this list.

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:50]: As with Ceph, Kubernetes should be removed from all of the items in this list.

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:69]:
```suggestion
### Nodes
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:74]:
```suggestion
### Postgres
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:78]:
```suggestion
### Prometheus
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:82]:
```suggestion
### Use Method
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:83]: As with Ceph, USE Method should be removed from all of the items in this list.

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:11]:
```suggestion

## Dashboards
```

File: operations/system_management_health/Grafana_Dashboards_by_Component.md
Comment (mharding-hpe) [inline operations/system_management_health/Grafana_Dashboards_by_Component.md:83]:
```suggestion

- Cluster
```

File: troubleshooting/index.md
Comment (mharding-hpe) [inline troubleshooting/index.md:10]:
```suggestion
```

File: troubleshooting/index.md
Comment (mharding-hpe) [inline troubleshooting/index.md:11]:
```suggestion
```

File: troubleshooting/index.md
Comment (mharding-hpe) [inline troubleshooting/index.md:12]:
```suggestion
```

File: troubleshooting/index.md
Comment (mharding-hpe) [inline troubleshooting/index.md:13]:
```suggestion
```

File: troubleshooting/index.md
Comment (mharding-hpe) [inline troubleshooting/index.md:15]:
```suggestion
 * [Grafana Dashboards](../operations/system_management_health/Grafana_Dashboards_by_Component.md)
    * [Troubleshoot Grafana Dashboard](../operations/system_management_health/Troubleshoot_Grafana_Dashboard.md)
 * Troubleshoot Kubernetes
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
In addition to the changes requested, this needs to be rebased again. I will want to re-review it after the rebase and the changes have been addressed. Also, once this does have the necessary approvals, we will need a backport PR to main (also also 1.0, if applicable)

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
As before, I will want to re-review before this merges.

**Review by mharding-hpe — APPROVED** [review]

**Comment by mharding-hpe** [comment]
I created the main branch backport for this: https://github.com/Cray-HPE/docs-csm/pull/1858
If this PR applies to csm-1.0, need a backport there.

**Review by rustydb — APPROVED** [review]

**Pull request merged**
